chevah_build

```diff
@@ -323,17 +323,13 @@ case $OS in
         export MAKE="make -P"
         export BUILD_LIBFFI="yes"
         export BUILD_ZLIB="yes"
+        # CFFI/cryptography build on HP-UX, but libffi breaks for CFFI. More at
```

Comment:
Maybe
CFFI/cryptography can be built/compiled on HP-UX but is not usable.

Reply:
Maybe… But  my message also conveys that it's because of libffi, so I still prefer it.